src/pages/Dashboard/Withdrawal.tsx

```
@@ -0,0 +1,161 @@
+    availableBalance: 15000,
+    accountStatus: "I",
+  },
+];
+
+const Withdrawal = () => {
+  const [step, setStep] = useState(1);
+  // const [page, setPage] = useState(1);
+  const [withdrawalMethod, setWithdrawalMethod] = useState(""); 
+  const [cardData, setCardData] = useState<any>([]);
+	const [selectedCard, setSelectedCard] = useState<any | undefined>({});
+  const [isTokenComplete, setIsTokenComplete] = useState(false);
```

ESLint

[error] 'isTokenComplete' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)